src/lsdb/catalog/catalog.py

```diff
@@ -1,10 +1,20 @@
-from typing import Dict
+from __future__ import annotations
+
+import dataclasses
+from typing import Dict, Tuple, Type
 
 import dask.dataframe as dd
 import hipscat as hc
 from hipscat.pixel_math import HealpixPixel
 
 from lsdb.catalog.dataset.dataset import Dataset
+from lsdb.core.crossmatch.abstract_crossmatch_algorithm import (
+    AbstractCrossmatchAlgorithm,
+)
+from lsdb.core.crossmatch.crossmatch_algorithms import BuiltInCrossmatchAlgorithm
+from lsdb.dask.crossmatch_catalog_data import (
```

Comment:
This looks like <110 chars. Re-run black.

Reply:
So this is what my black running does, is there any other config I should be setting?

Reply:
Ah. You should also run `isort`. That will change at least 6 files (including this line).

Reply:
isort seemed to fix it, thanks